pantheon-sessions.php

```diff
@@ -452,7 +452,7 @@ protected function safe_output( $message, $type, array $variables = [] ) {
 			return;
 		}
 
-		print "\n" . vsprintf( $message, $variables );
+		print "\n" . esc_html( vsprintf( $message, $variables ) );
```

Comment:
This will strip HTML (or break HTML) in any messages sent to `safe_output`. If you haven't already, you should check all the places where `safe_output` is used to see if they would be affected by stripping the html out. If so, I might recommend using `wp_kses_post` intead, which sanitizes the output but allows HTML that would typically be included in a post.